runtime/plugin/extra-keymaps.vim

```diff
@@ -0,0 +1,44 @@
+" Extra default keymaps and keyboard behavior
+" Maintainer: Felipe Morales <hel.sheep@gmail.com>
```

Comment:
Not sure about this. I'd rather we keep with what has been done so far, which is to not cite a specific maintainer. Ownership (or exclusive knowledge) over a specific area code is something we shouldn't adopt IMO -- I've heard of at least a few cases where maintainers went missing so various runtime files couldn't be updated.